teku/src/main/java/tech/pegasys/teku/services/ServiceController.java

```diff
@@ -37,7 +38,8 @@ public ServiceController(final ServiceConfig config) {
     if (config.getConfig().isRemoteValidatorApiEnabled()) {
       services.add(new RemoteValidatorService(config));
     } else {
-      services.add(ValidatorClientService.create(config));
+      final Path slashingProtectionPath = config.getConfig().getValidatorsSlashingProtectionPath();
+      services.add(ValidatorClientService.create(config, slashingProtectionPath));
```

Comment:
We can just pass in `config` now and have `ValidatorClientService` get the slashing protection path from the config itself.